custom-widget-builder/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "custom-widget-builder",
```

Comment:
I think it could be better to put @bonitasoft/custom-widget-builder as name. This prefix will be put this library in bonitasoft namespace.

Reply:
ok